WeatherPage.xaml

```diff
@@ -9,21 +9,23 @@
     <ScrollView>
         <ScrollView>
             <VerticalStackLayout
-            Spacing="25"
-            Padding="30,0"
             VerticalOptions="Start">
                 <!-- This line says that the ItemsSource of the CollectionView is the Weathers property of the Domain object (an ObservableCollection) -->
-                <CollectionView x:Name="CV" ItemsSource="{Binding Weathers}" SelectionMode="Single">
+                <CollectionView x:Name="CV" ItemsSource="{Binding ClosestAirportWeather}" SelectionMode="Single">
                     <CollectionView.ItemTemplate>
                         <DataTemplate x:DataType="model:Weather">
-                            <Grid RowDefinitions="*,*,*,*" ColumnDefinitions="*">
-                                <Label Grid.Row="0" Text="METAR:" VerticalTextAlignment="Center" FontSize="32" FontAttributes="Bold" Margin="20,10,0,10"/>
-                                <Frame Grid.Row="1" Margin="20" CornerRadius="5" Padding="5">
-                                    <Label Grid.Row="1" Text="{Binding Metar}" HorizontalOptions="Center" VerticalTextAlignment="Center" FontSize="24" Margin="0,10,0,10"/>
+                            <Grid Margin="16" RowDefinitions="*,*,*,*,*,*,*,*,*" ColumnDefinitions="*">
+                                <Label Grid.Row="0" Text="Closest Airport:" VerticalTextAlignment="Center" FontSize="32" FontAttributes="Bold" Margin="0,10,0,0"/>
```

Comment:
Upon reflection, in the interests of consistency with the other pages, let's just delete this label, leaving Weather in the navigation bar.